src/filter/mod.rs

```diff
@@ -341,6 +343,50 @@ pub trait Filter: FilterBase {
         UntupleOne { filter: self }
     }
 
+    /// Wraps up the extracted arguments into a tuple.
+    ///
+    /// This is the inverse of `untuple_one`. It is mainly useful in generic code
+    /// that works on arbitrary filters, without knowing how many arguments the
+    /// filter has extracted.
+    ///
+    /// # Example
+    ///
+    /// ```
```

Comment:
I'd worry the stuff about `spawn_blocking` distracts from the purpose of the method. How about just a simple example combining multiple path extractions. Like:

```rust
let filter = warp::path!(u32 / String);

// Normally
filter.map(|n: u32, s: String| format!("{}: {}", n, s));

// With tuple_args
filter.tuple_args().map(|tup: (u32, String)| format!("{:?}", tup));
```